### DISCUSSION TITLE
fromitive-blog/project/2023-07-17-backtest/

### DISCUSSION THREAD

**Discussion by giscus** [fromitive/fromitive-blog#16] [Q&A]
# fromitive-blog/project/2023-07-17-backtest/

bithumb API를 사용해보자

https://fromitive.github.io/fromitive-blog/project/2023-07-17-backtest/

**Comment by DDONONG** [comment]
대단해요‼️

**Comment by Lousisi1** [comment]
좋은 포스팅 감사합니다.
같은 문제로 고통받고 있는데, 올려주신 방법대로 해도 bad request 에러가 뜨네요 ㅠㅠ 
현재도 동일한 헤더로 사용하고 계실까요?
